sys/arm64/arm64/cpufunc_asm.S

```diff
@@ -70,7 +66,16 @@ __FBSDID("$FreeBSD$");
 	add	x1, x1, x2		/* Add these to the size */
 	bic	x0, x0, x4		/* Clear the low bit of the address */
 #ifdef __CHERI_PURE_CAPABILITY__
-	scvalue	c0, c6, x0
+	/*
+	 * There is no guarantee that the capability given here has
```

Comment:
Doing this without any checks for ivac specifically scares me from a memory safety perspective; the others are fine.

Reply:
Should be fixed now.